hubitat.py

```diff
@@ -20,7 +23,13 @@ def __init__(self, conf: dict):
         # because Python doesn't support case insensitive searches
         # and Hubitats requires exact case, we create a dict{lowercase,requestedcase}
         self.hsm_arm: dict[str, str] = {x.lower(): x for x in conf["hsm_arm_values"]}
-        for name, data in conf["device_groups"].items():
+        enabled_device_groups = conf["enabled_device_groups"]
+        if not enabled_device_groups:
```

Comment:
Is it ever going to work?  This dictionary lookup will raise KeyError.  Maybe you want `.get()` ?

Reply:
it's always going to work, because template.config.yaml is always read and has:
`  enabled_user_groups: [ ]         # List of enabled user groups. If empty, none are enabled.`

The check is to catch cases where this setting is not getting overridden